packages/modules/viewer-react/src/components/error/ErrorBoundary.tsx

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import type { ErrorPageProps, ErrorPageType } from "@itwin/itwinui-react";
+import { ErrorPage } from "@itwin/itwinui-react";
+import React, { Component } from "react";
+
+interface Props {
+  eventTitle?: string;
+}
+
+interface Error {
```

Comment:
do we need this type? isnt this built into typescript?

Reply:
It is, but TypeScript's Error interface doesn't have a field for the error code. Let me see if it still works if I revert it.

Reply:
but are you even using error code?